### PR TITLE
Sycamore tableau function

### PR DESCRIPTION
**Pull request by clementzach**
This creates a convenience function for generating survey tableau summaries

### CODE REVIEW

**Review by Noah-Houghton — COMMENTED** [review]
Overall functionality looks fine to me; 1 style / design question and 1 logic question.

File: forest/sycamore/base.py [hunk 1]
```diff
@@ -169,3 +170,87 @@ def compute_survey_stats(
...
+    Returns:
+    Writes a csv file for each user in the output folder with survey summary
+        statistics"""
+    os.makedirs(output_folder, exist_ok=True)
+    if users is None:
+        users = get_ids(study_folder)
+    if end_date is None:
+        end_date = get_month_from_today()
+    # Read, aggregate and clean data
+    if config_path is None:
```
[the remark this answers is not in the document]
Comment (clementzach) [reply forest/sycamore/base.py:216]: Good point. Yes, it doesn't make sense to run this without a config file. I'll update that

File: forest/sycamore/base.py [hunk 2]
```diff
@@ -169,3 +170,87 @@ def compute_survey_stats(
...
+        )
+        if agg_data.shape[0] == 0:
+            logger.error("Error: No survey data found in %s", study_folder)
+            return True
+        # Create survey submits detail and summary
+        ss_detail = survey_submits(
+            config_path, start_date, end_date,
+            users, agg_data, interventions_filepath
+        )
+        if ss_detail.shape[0] == 0:
```
[the remark this answers is not in the document]
Comment (clementzach) [reply forest/sycamore/base.py:229]: This was relevant in compute_survey_stats, but not in this function. Thanks for noticing!

[the remark this answers is not in the document]
Comment (clementzach) [reply forest/sycamore/base.py:227]: @Noah-Houghton if this code looked good to you, would you mind approving the review?

**Comment by clementzach** [comment]
> Looks fine to me - if you want a second opinion feel free to solicit one (since it's been a while since I looked at this) but nothing stands out as incorrect, seems like it does what it sets out to do.

Awesome, thank you, @Noah-Houghton !

**Review by hackdna — APPROVED** [review]

**Pull request merged**